.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
```

Comment:
Also link to [contributing guidelines](https://github.com/freeCodeCamp/devdocs/blob/master/.github/CONTRIBUTING.md)?

Reply:
I do not think it is needed to add the contributing guidelines, the templates already have most of the information found in the guidelines and the information about how contributing is not needed to add make a new issue.